Marlin/language_de.h

```diff
@@ -153,6 +159,12 @@
 #define MSG_ERR_MINTEMP                     LCD_STR_THERMOMETER " UNTERSCHRITTEN"
 #define MSG_ERR_MAXTEMP_BED                 "BETT " LCD_STR_THERMOMETER " ÜBERSCHRITTEN"
 #define MSG_ERR_MINTEMP_BED                 "BETT " LCD_STR_THERMOMETER " UNTERSCHRITTEN"
+#define MSG_ERR_Z_HOMING                    "G28 Z Forbidden"
```

Comment:
That too. I placed the string in `language_en.h` with the intent that it should be translated.